in_toto/keylib.go

```diff
@@ -352,6 +356,9 @@ func (k *Key) LoadKeyReader(r io.Reader, scheme string, keyIdHashAlgorithms []st
 		if err := k.setKeyComponents(pubKeyBytes, []byte{}, ecdsaKeyType, scheme, keyIdHashAlgorithms); err != nil {
 			return err
 		}
+	case *x509.Certificate:
+		k.KeyVal.Certificate = strings.TrimSpace(string(pemBytes))
```

Comment:
This is pretty gross, and is fragile.  It should live in setKeyComponents, but changing the signature of that function was a lot.  Will address this in refactor.

Reply:
:see_no_evil: